build

```diff
@@ -0,0 +1,18 @@
+<Project>
```

Comment:
Is this file supposed to have a file extension? I should think so, no?

Reply:
Does not need one. Can make it "build.cmd" instead if that's better?

Reply:
To run it we will need to do "msbuild .\build" or "msbuild .\build.cmd" (if we add the .cmd extension)